### DISCUSSION TITLE
Multi-day booking?

### DISCUSSION THREAD

**Discussion by DarrianCZE** [daveroverts/bmac#862] [General]
Hi. As I was exploring the event, I have noticed that it doesn't allow the event to take more than one day (since you are not able to specify the date of added flights.

Is there any chance to implement this in the future?

We are organizing a real ops event, but some flights start at a day before due to timezone differences, and therefore we are unable to add those flights. Many thanks!

**Comment by daveroverts** [comment]
Are you using both `ctot` and `eta` values in the same flight? Events on Vatsim usually don't use both values as there's a difference between a B77W, A320 and DH8D. That's the reason the forms + import just uses the event date.

I could add support for it when I have some time available. In the meantime, you could in theory change values directly in the database. If you setup email, keep in mind that the [Booking confirmed email](https://github.com/daveroverts/bmac/blob/ae6ba09241cb38f2e342954a790fd4d410a4d751/resources/views/emails/booking/confirmed.blade.php#L41) shows the event date. In both cases, I recommend adding a note that the pilot should depart the day before.

**Comment by DarrianCZE** [comment]
Yes, we are. We are having a "hardcore" real ops event with pre-defined aircraft types, so ctot and eta are both valid. Thanks for the tip!
